### ISSUE TITLE
Issue with the alignment of header of PDF export of reports

### ISSUE THREAD

**Issue by kdjstudios** [kdjstudios/SABillingGitlab#1148]
In GitLab by @pchaudhary on Sep 28, 2018, 06:37

There is an existing issue with all of the reports that PDF export headers on all pages except the first are not aligned correctly.

**Comment by kdjstudios** [comment]
In GitLab by @pchaudhary on Sep 28, 2018, 06:40

Hi [Kyle]  ( @kdjstudios ),  

We looked at the dev server & found that on dev reports there are only headers for the first page. 
We also would like to ask: should we go with the dev approach?

Thanks.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Sep 28, 2018, 11:27

@pchaudhary 

Thanks for opening a separate ticket. I just want to clarify you are stating this is an issue on all servers, not just dev. 

Also this must be fixed before we can release #1090 as for their requested reports to be working correctly will need to have the PDF export option working correctly.

**Comment by kdjstudios** [comment]
In GitLab by @pchaudhary on Sep 28, 2018, 23:58

@kdjstudios 

> Thanks for opening a separate ticket. I just want to clarify you are stating this is an issue on all servers, not just dev.

* Yes, this is an ongoing issue on all of the servers with all of the reports. But only just Dev, has correct values.



> Also this must be fixed before we can release #1090 as for their requested reports to be working correctly will need to have the PDF export option working correctly.

* Yes, we know that this must be resolved along with #1090 but we still looking for the reply on : `should we go with the dev approach?`, which means that should we go with the approach of **dev reports in which the headers are only on the first page.**?  

Thanks.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Sep 29, 2018, 12:47

@pchaudhary 

Please bring this up for discusions with @tim.traylor. I would think we should have the headers display on all pages, otherwise it will be very difficult to know the difference between values if your not on the first page.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jan 29, 2019, 14:09

Marking this for secondary to #1324

**Comment by kdjstudios** [comment]
In GitLab by @amishra on Feb 8, 2019, 05:35

@kdjstudios 

Thank you for the update.

This was completed under #1324 ticket and ready for production.

Please update the label.

thanks

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Feb 8, 2019, 08:56

@amishra 

Thank you. Marking as ready.

**Comment by kdjstudios** [comment]
In GitLab by @pchaudhary on Feb 16, 2019, 04:48

@kdjstudios 

We have successfully released on both of the production servers.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Feb 18, 2019, 13:34

closed
